### PR TITLE
Turn `@method` annotations into real method declarations

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 6.0
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Component/Translation/TranslatorBagInterface.php [hunk 1]
```diff
@@ -30,4 +26,11 @@ interface TranslatorBagInterface
      * @throws InvalidArgumentException If the locale contains invalid characters
      */
     public function getCatalogue(string $locale = null): MessageCatalogueInterface;
+
+    /**
+     * Returns all catalogues of the instance.
```
Comment (wouterj) [inline src/Symfony/Component/Translation/TranslatorBagInterface.php:31]: I think we can safely remove this short description, it doens't add anything.

File: src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php [hunk 2]
```diff
@@ -20,15 +20,13 @@
...
-use Symfony\Component\Routing\RouteCollectionBuilder;
 
 /**
  * A Kernel that provides configuration hooks.
  *
  * @author Ryan Weaver <ryan@knpuniversity.com>
  * @author Fabien Potencier <fabien@symfony.com>
  *
- * @method void configureRoutes(RoutingConfigurator $routes)
  * @method void configureContainer(ContainerConfigurator $container)
```
Comment (wouterj) [inline src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php:30]: Didn't you forget this one?

Comment (nicolas-grekas) [reply src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php:30]: we also accept this (and did not deprecate it to not force using the DSL for the sake of it):
`configureContainer(ContainerBuilder, LoaderInterface)`

File: src/Symfony/Component/Translation/TranslatorBagInterface.php (hunk 1 above)
Comment (nicolas-grekas) [reply src/Symfony/Component/Translation/TranslatorBagInterface.php:31]: on contracts it's worth being extra clear I think

**Pull request merged**